drivers/python/rethinkdb/net_twisted.py

```diff
@@ -51,10 +40,6 @@ def __init__(self, factory):
 
         self._open = True
 
-    def resetBuffer(self):
-        self.buf = ''
-        self.buf_expected_length = 0
-
```

Comment:
We never actually want to throw away what's in the buffer - it should carry over from one stage to the next.